azure-pipelines.yml

```diff
@@ -19,7 +19,7 @@ pr:
 variables:
   # Path where packages (nuget or app packages) will be copied to.
   PackageOutputPath: $(Build.ArtifactStagingDirectory)
-  XCODE_ROOT: '/Applications/Xcode_15.2.app'
+  XCODE_ROOT: '/Applications/Xcode_16.app'
```

Comment:
I see the build is failing because it's not finding Xcode.

Would it be Xcode_16.2 ? (The latest version before 16.3)

Reply:
Yeah I'm testing out fixes in the canaries/dev branch and then I will redo the merge :)